src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { node } from 'prop-types';
+import GlobalStyle from '@bbc/psammead-styles/global-styles';
```

Comment:
For consistency with other usage of this import. 
```suggestion
import GlobalStyles from '@bbc/psammead-styles/global-styles';
```